app/views/curation_concerns/file_sets/show.html.erb

```diff
@@ -5,6 +5,7 @@
     <div class="col-xs-12 col-sm-4">
       <%= media_display @presenter %>
       <%= render 'show_actions', presenter: @presenter %>
+      <%= render 'single_use_links', presenter: @presenter %>
```

Comment:
So are we showing all single-use links to everybody who can get to the show view, not just the folks who have manage/edit access to the file_set?